tests/cases/distributed/information_schema/region_peers.result

```diff
@@ -0,0 +1,58 @@
+--- test information_schema.region_peers ----
+USE INFORMATION_SCHEMA;
+
+Affected Rows: 0
+
+CREATE TABLE region_peers_phy (ts timestamp time index, val double) engine = metric with ("physical_metric_table" = "");
+
+Affected Rows: 0
+
+CREATE TABLE region_peers_t1 (
+    ts timestamp time index,
+    val double,
+    host string primary key
+) engine = metric with ("on_physical_table" = "region_peers_phy");
+
+Affected Rows: 0
+
+CREATE TABLE region_peers_t2 (
+    ts timestamp time index,
+    job string primary key,
+    val double
+) engine = metric with ("on_physical_table" = "region_peers_phy");
+
+Affected Rows: 0
+
+CREATE TABLE region_peers_test (
+    a int primary key,
+    b string,
+    ts timestamp time index,
+) PARTITION ON COLUMNS (a) (
+    a < 10,
+    a >= 10 AND a < 20,
+    a >= 20,
+);
+
+Affected Rows: 0
+
+SELECT region_id,is_leader,status,down_seconds FROM region_peers ORDER BY region_id;
```

Comment:
Oh, my bad. It's hard to test for these cases. The RegionId may be changed due to the execution being in a different order. How about changing SQL to `SELECT count(distinct region_id) FROM region_peers ORDER BY region_id;`, then we can move this test into `common` dir.